src/styles/nav.scss

```diff
@@ -1,41 +1,54 @@
-nav {
-    min-height: 10vh;
+nav.main-nav {
```

Comment:
Same here as the footer. Your nav component appears on every page and there may be times when you use another <nav> element. You want to make sure you're only targeting this specific nav.